internal/nimparser/nimparser.go

```diff
@@ -166,6 +177,10 @@ func MatchProfiles(modelSpec appsv1alpha1.ModelSpec, manifest NIMManifest, disco
 	return selectedProfiles, nil
 }
 
+func isOptimizedEngine(engine string) bool {
+	return engine != "" && strings.Contains(strings.ToLower(engine), BackendTypeTensorRT)
```

Comment:
Are we going to have additional backend types in the future? If yes please add todo....

Reply:
I am not sure on that, we can add if NVIDIA builds for other backends.